File: pset1/cash/cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>

int main(void)
{
    float cash;
    int cents, quarters, dimes, nickels, pennies, coins;
    do
    {
        cash = get_float("Change owed: ");// Get coins
    }
    while(cash < 0); // Ensure that the value entered is non-negative
    cents = round(cash*100); //Print cash into cents

    quarters = cents/25; // Get number of quarters
    dimes = (cents%25)/10; // Get number of dimes
    nickels = (cents%25%10)/5; // Get number of nickels
    pennies = (cents%25%10%5)/1; // Get number of pennies

    coins = quarters + dimes + nickels + pennies; // Sum total number of coins

    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

# Get float that is not negative
while True:
    change = get_float("Change owed: ")
    if change >= 0:
        break

cents = round(change * 100)

quarters = int(cents / 25)
dimes = int((cents % 25) / 10)
nickels = int((cents % 25 % 10) / 5)
pennies = int((cents % 25 % 10 % 5) / 1)

coins = quarters + dimes + nickels + pennies

print(f"{coins}")